manila/share/drivers/netapp/dataontap/cluster_mode/lib_base.py

```diff
@@ -4220,6 +4220,14 @@ def ensure_shares(self, context, shares):
                 else:
                     LOG.warning(msg)
 
+        try:
+            self._client.prune_deleted_nfs_export_policies()
+            self._client.prune_deleted_snapshots()
+            self._client.prune_deleted_volumes()
+        except Exception as e:
+            LOG.warning("Failed to cleanup resources in ensure share"
```

Comment:
```suggestion
            LOG.warning("Failed to cleanup resources in ensure share: "
```

I suggest to add some separator between `share` and `Error` in the warning log message